src/frontend/js/conversazioni.js

```diff
@@ -0,0 +1,109 @@
+import { LitElement, html, css } from "lit";
+
+class Conversazioni extends LitElement {
+
+    static properties = {
+        elencoConversazioni: {state: true},
```

Comment:
usiamo l'inglese per i nomi di classi, funzioni o variabili

Reply:
cambiato in chatsList